birdhouse/optional-components/wps-healthchecks/config/catalog/docker-compose-extra.yml

```diff
@@ -1,15 +0,0 @@
-version: "3.4"
```

Comment:
Shouldn't this be under `birdhouse/optional-components/wps-healthchecks/config/catalog/docker-compose-extra.yml`? 
It is now in `birdhouse/optional-components/wps-healthchecks/catalog/docker-compose-extra.yml` (missing nested `/config/`)?